galleries/users_explain/figure/interactive.rst

```diff
@@ -325,25 +326,15 @@ Interactive mode works in the default Python prompt:
    >>> plt.ion()
    >>>
 
-however this does not ensure that the event hook is properly installed
+However, this does not ensure that the event hook is properly installed
 and your figures may not be responsive.  Please consult the
 documentation of your GUI toolkit for details.
 
 
 .. _jupyter_notebooks_jupyterlab:
 
 Jupyter Notebooks / JupyterLab
-------------------------------
-
-.. note::
-
-   To get the interactive functionality described here, you must be
-   using an interactive backend.  The default backend in notebooks,
-   the inline backend, is not.  `~ipykernel.pylab.backend_inline`
-   renders the figure once and inserts a static image into the
-   notebook when the cell is executed.  Because the images are static, they
-   cannot be panned / zoomed, take user input, or be updated from other
-   cells.
+==============================
```

Comment:
I disagree with changing the level of the headings here, I think Jupyter is actually intended to be underneath the `Other Python Prompts` banner... And now it looks like `Spyder/Other IDEs` is underneath `Jupyter Notebooks`, which is just incorrect.